Feature/debounce

joshinehajoshi opened the pull request:
DEBOUNCE_CONST added for 500 milisecond

awesomearpit commented:
Hi @BandanaPandey, Thanks for getting interested in dailyploy-web. Actually when I marge your branch than I had to face a conflict issue. So I applied your changes in my existing code and push the code.
So I am closing your pull request.

amolbhosale11 committed:
sign form design  changes

amolbhosale11 committed:
sign up page design changes

amolbhosale11 committed:
changes design form left and right side images

amolbhosale11 committed:
sign in and sign up pages resdesing changes

amolbhosale11 committed:
signin and up form design changes done

amolbhosale11 committed:
user settings dashboard upload image added

amolbhosale11 committed:
hovet effect added in buttons

amolbhosale11 committed:
workspace UI modal centerd and hover added

amolbhosale11 committed:
hover btn and filter color changed

amolbhosale11 committed:
hover filter color added

Commit:
changes nded for notication

amolbhosale11 committed:
workspace tooltip position fixed

amolbhosale11 committed:
task added

amolbhosale11 committed:
add task in comment section redesign changes

amolbhosale11 committed:
comment section design WIP

amolbhosale11 committed:
comment background color

amolbhosale11 committed:
add dropdown in admin section design chnaged of opening dropdown

amolbhosale11 committed:
commnent section name and comment name color changed

amolbhosale11 committed:
comment section submit on enter press key

amolbhosale11 committed:
comment section in comment input box changes

amolbhosale11 committed:
comment section div implement

amolbhosale11 committed:
comment box some attachement changes

amolbhosale11 committed:
comments text remove and user circle initial added

amolbhosale11 committed:
fix member and project icon on serach bar and edit mail hover issue fixed

Commit:
Render method method in Timer

Commit:
Eliminated warnings
- Removed Class with className
- Removed hover with hover, focus

Commit:
- Remobved state update from did mount lifecycle metrhod

makstackavenue9456 committed:
Merge branch 'feature/remove-frequeny-reports' into 'development'

Feature/remove frequeny reports

See merge request stackavenue/dailyploy-web!51

makstackavenue9456 committed:
Idle Screen breaking fix
- Returned a NA value instead of empty value

makstackavenue9456 committed:
Idle Screen breaking fix
- Returned a NA value instead of empty value

makstackavenue9456 committed:
Merge branch 'feature/remove-frequeny-reports' into 'development'

Feature/remove frequeny reports

See merge request stackavenue/dailyploy-web!52

makstackavenue9456 committed:
Fix issue/task card html

makstackavenue9456 committed:
Merge branch 'fixIssue/task-card-html' into 'development'

Fix issue/task card html

See merge request stackavenue/dailyploy-web!53

Commit:
- Added loader for DailyPloy
- Added right margin on clock

makstackavenue9456 committed:
Merge branch 'bug/Dashboard-UI-lag' into 'development'

Bug/dashboard ui lag

See merge request stackavenue/dailyploy-web!54

makstackavenue9456 committed:
- Added Purecomponent to Dashboard js
- Added null check with componentdidmount

Commit:
[story_135] Fix loader issue in project show page

makstackavenue9456 committed:
Merge branch 'issues/story_135-issues' into 'development'

[story_135] Fix loader issue in project show page

See merge request stackavenue/dailyploy-web!55

makstackavenue9456 committed:
- label changes
- task list changes

makstackavenue9456 committed:
Merge branch 'fixIssue/task-card-html' into 'development'

Fix issue/task card html

See merge request stackavenue/dailyploy-web!56

makstackavenue9456 committed:
Merge branch 'fixIssue/roadmap-hotfixv1' into 'development'

Fix issue/roadmap hotfixv1

See merge request stackavenue/dailyploy-web!58

makstackavenue9456 committed:
- label changes
- task list changes

makstackavenue9456 committed:
- Added componentShouldUpdate on taskInfoModal to avoid renrender and added cond to load Task Info only when clicked
- Removed Table Warning

makstackavenue9456 committed:
Merge branch 'fixIssue/performance-dashboard' into 'development'

Fix issue/performance dashboard

See merge request stackavenue/dailyploy-web!59

makstackavenue9456 committed:
Merge branch 'fixIssue/Dashboard-hang-Issue' into 'development'

Fix issue/dashboard hang issue

See merge request stackavenue/dailyploy-web!61

makstackavenue9456 committed:
- Added componentShouldUpdate on taskInfoModal to avoid renrender and added cond to load Task Info only when clicked
- Removed Table Warning

makstackavenue9456 committed:
- label changes
- task list changes

makstackavenue9456 committed:
Merge branch 'bugfix/roadmap' into 'development'

Bugfix/roadmap

See merge request stackavenue/dailyploy-web!68

makstackavenue9456 committed:
Merge branch 'fix/Multiple-task-card' into 'development'

Fix/multiple task card

See merge request stackavenue/dailyploy-web!69

makstackavenue9456 committed:
- CHanges rverted for Multiple task on Dashboard
- Placeholder change on Estimated time

makstackavenue9456 committed:
Merge branch 'features/arrows-between-same-tasks' into 'development'

Features/distinguish same tasks

See merge request stackavenue/dailyploy-web!70

Jeshan7 committed:
- Added componentShouldUpdate on taskInfoModal to avoid renrender and added cond to load Task Info only when clicked
- Removed Table Warning

Jeshan7 committed:
Removed opacity from non-selected task on dashboard

makstackavenue9456 committed:
Merge branch 'fixissue/remove-opacity' into 'development'

Removed opacity from non-selected task on dashboard

See merge request stackavenue/dailyploy-web!72

makstackavenue9456 committed:
Merge branch 'feature/estimatedtime-taskstatus' into 'development'

Task status and estimated time of tasks added in Reports table (ReportTableRow cmp)

See merge request stackavenue/dailyploy-web!71

iphalgun committed:
Task status and estimated time of tasks added in Reports table (ReportTableRow cmp)

makstackavenue9456 committed:
- Added label hour in Reports Table

makstackavenue9456 committed:
Merge branch 'fix/label-change' into 'development'

- Added label hour in Reports Table

See merge request stackavenue/dailyploy-web!73

geektj committed:
Edit/delete time in Task details

makstackavenue9456 committed:
Merge branch 'bugfix/Dashboard-TaskDetails' into 'rc-v1.7.6'

Edit/delete time in Task details

See merge request stackavenue/dailyploy-web!75

makstackavenue9456 committed:
Merge branch 'feature/taskCard-estTime_hover-timeInSec' into 'rc-v1.7.6'

Feature/task card est time hover time in sec

See merge request stackavenue/dailyploy-web!76

iphalgun committed:
-Hovering on task card shows estimated time now, changes made in Calender and Dashboard cmp.
-Task card shows time in seconds now, changes made in Dashboard, DashboardEvent and TimeCardTask cmps.

makstackavenue9456 committed:
- Changes to show Task edit pencil

Jeshan7 committed:
- Improved UI of stretching cards in dashboard
  - Fixed clock size

makstackavenue9456 committed:
Merge branch 'feature/modal-on-exit' into 'rc-v1.7.6'

Fixissues/improve-stretching-UI

See merge request stackavenue/dailyploy-web!77

iphalgun committed:
-Hovering on task card shows estimated time now, changes made in Calender and Dashboard cmp.
-Task card shows time in seconds now, changes made in Dashboard, DashboardEvent and TimeCardTask cmps.

makstackavenue9456 committed:
Merge branch 'feature/default-status-priority' into 'rc-v1.7.6'

Feature/default status priority

See merge request stackavenue/dailyploy-web!79

makstackavenue9456 committed:
Merge branch 'rc-v1.7.6' of https://gitlab.com/stackavenue/dailyploy-web into rc-v1.7.6

makstackavenue9456 committed:
Merge branch 'rc-v1.7.6' into 'development'

Rc v1.7.6

See merge request stackavenue/dailyploy-web!80

makstackavenue9456 committed:
-Hovering on task card shows estimated time now, changes made in Calender and Dashboard cmp.
-Task card shows time in seconds now, changes made in Dashboard, DashboardEvent and TimeCardTask cmps.

geektj committed:
- project heading show vertical to horizontal

geektj committed:
- users does not select date show starts and ends

geektj committed:
- create roadmap from or to date not selected show in task space blank

geektj committed:
Merge branch 'development' of gitlab.com:stackavenue/dailyploy-web into bugfix/UI_Roadmap

geektj committed:
- shows date in roadmap start and end date format change

makstackavenue9456 committed:
Merge branch 'fix/production-hotfix' into 'development'

Fix/production hotfix

See merge request stackavenue/dailyploy-web!83

makstackavenue9456 committed:
Fix/production hotfix

makstackavenue9456 committed:
-Hovering on task card shows estimated time now, changes made in Calender and Dashboard cmp.
-Task card shows time in seconds now, changes made in Dashboard, DashboardEvent and TimeCardTask cmps.

makstackavenue9456 committed:
Merge branch 'feature/task-status-labels' into 'development'

Feature/task status labels

See merge request stackavenue/dailyploy-web!84

makstackavenue9456 committed:
- Mark completed success toast and hide the success box
- Changed cond in Dashboard Event to render on basis of is_complete

makstackavenue9456 committed:
Merge branch 'feature/mark-as-completed' into 'development'

Feature/mark as completed

See merge request stackavenue/dailyploy-web!85

Jeshan7 committed:
Added Project monthly option on add project modal and edit project modal

Jeshan7 committed:
Merge branch 'development' of https://gitlab.com/stackavenue/dailyploy-web into feature/add-monthly-budget

iphalgun committed:
-Hovering on task card shows estimated time now, changes made in Calender and Dashboard cmp.
-Task card shows time in seconds now, changes made in Dashboard, DashboardEvent and TimeCardTask cmps.

makstackavenue9456 committed:
Merge branch 'feature/filter-roadmaps' into 'development'

Feature/filter roadmaps

See merge request stackavenue/dailyploy-web!87

iphalgun committed:
-Hovering on task card shows estimated time now, changes made in Calender and Dashboard cmp.
-Task card shows time in seconds now, changes made in Dashboard, DashboardEvent and TimeCardTask cmps.

makstackavenue9456 committed:
Merge branch 'feature/filter-roadmaps' into 'development'

Feature/filter roadmaps

See merge request stackavenue/dailyploy-web!89

makstackavenue9456 committed:
- Mark completed success toast and hide the success box
- Changed cond in Dashboard Event to render on basis of is_complete

makstackavenue9456 committed:
Merge branch 'feature/mark-as-completed' into 'development'

Feature/mark as completed

See merge request stackavenue/dailyploy-web!90

makstackavenue9456 committed:
Merge branch 'feature/add-monthly-budget' into 'development'

Feature/add monthly budget

See merge request stackavenue/dailyploy-web!86

Jeshan7 committed:
Feature/add monthly budget

makstackavenue9456 committed:
Merge branch 'feature/milestone' into 'development'

Feature/milestone

See merge request stackavenue/dailyploy-web!92

makstackavenue9456 committed:
- Added background image on 0 milestone
- Added debounce on add, edit and delete function

makstackavenue9456 committed:
Merge branch 'feature/roadmap-status' into 'development'

Feature/roadmap status

See merge request stackavenue/dailyploy-web!91

iphalgun committed:
-Roadmap task cards now have a status dropdown.
-Dropdown added next to task card name.

makstackavenue9456 committed:
Merge branch 'development' of gitlab.com:stackavenue/dailyploy-web into bugfix/UI_Roadmap

makstackavenue9456 committed:
- CHanges relted to showing proper date formats in roadmap

makstackavenue9456 committed:
- CSS CHANGES FOR  RDmap text name and Date desplay

makstackavenue9456 committed:
Added no Timeline in case of no Dates selected

geektj committed:
-Hovering on task card shows estimated time now, changes made in Calender and Dashboard cmp.
-Task card shows time in seconds now, changes made in Dashboard, DashboardEvent and TimeCardTask cmps.

makstackavenue9456 committed:
Merge branch 'bugfix/UI_Roadmap' into 'development'

UI improve on roadmap

See merge request stackavenue/dailyploy-web!81

makstackavenue9456 committed:
Merge branch 'bugfix/UI_Roadmap' into 'development'

Bugfix/ui roadmap

See merge request stackavenue/dailyploy-web!94

makstackavenue9456 committed:
-- roadmap status name changes with font changes

makstackavenue9456 committed:
Merge branch 'fx/Costtic-fixes' into 'development'

Fx/costtic fixes

See merge request stackavenue/dailyploy-web!95

makstackavenue9456 committed:
Merge branch 'feature/milestone' into 'development'

Feature/milestone

See merge request stackavenue/dailyploy-web!96

makstackavenue9456 committed:
- Added background image on 0 milestone
- Added debounce on add, edit and delete function

iphalgun committed:
-Sign up and login page changed (old one)
-Top margin added to DailyPloy logo
-Cursor pointer added to password eye

Jeshan7 committed:
Merge branch 'development' of https://gitlab.com/stackavenue/dailyploy-web into feature/add-monthly-budget

Jeshan7 committed:
added per month string after budget input box

makstackavenue9456 committed:
Merge branch 'feature/oldLoginSignUp' into 'development'

Sign up and login page reverted to previous design

See merge request stackavenue/dailyploy-web!97

makstackavenue9456 committed:
Merge branch 'feature/add-monthly-budget' into 'development'

Feature/add monthly budget

See merge request stackavenue/dailyploy-web!98

makstackavenue9456 committed:
Merge branch 'feature/roadmap-checklist' into 'development'

Feature/roadmap checklist

See merge request stackavenue/dailyploy-web!93

iphalgun committed:
-Added Checklist UI in roadmaps.
-Two new cmps added, Checklist and ChecklistAddTask.

yasir-stack-avenue committed:
Error Modal added

makstackavenue9456 committed:
Merge branch 'feature/error-handle-dp' into 'development'

Error Modal added

See merge request stackavenue/dailyploy-web!100

iphalgun committed:
-Added loader to summary, task list tasks and task card
-Member restrictions added in Workspace settings
-Font size in filter dropdown has been fixed
-Removed create, delete edit roadmap icons for members.

makstackavenue9456 committed:
Merge branch 'fixIssues/roadmapFilter' into 'development'

UI Issues Fixed

See merge request stackavenue/dailyploy-web!99

iphalgun committed:
Settings section error 404 resolved

makstackavenue9456 committed:
Merge branch 'fixIssue/settings' into 'development'

Settings section error 404 resolved

See merge request stackavenue/dailyploy-web!104

iphalgun committed:
-Roadmap milestone gap issue fixed
-Milestone is now visible to admin only

makstackavenue9456 committed:
Merge branch 'fixIssue/milestoneAccess' into 'development'

-Roadmap milestone gap issue fixed

See merge request stackavenue/dailyploy-web!105

iphalgun committed:
-Resoure Allocation UI
-Created a new cmp, Allocation.js

makstackavenue9456 committed:
Merge branch 'feature/resoure-allocation' into 'development'

Feature/resoure allocation

See merge request stackavenue/dailyploy-web!106

Jeshan7 committed:
Same text occurs after  creating new category
 in addtask modal - fixed

makstackavenue9456 committed:
Merge branch 'bugfix/new-category' into 'development'

Category dailyploy select error

See merge request stackavenue/dailyploy-web!107

iphalgun committed:
-Added loader- add/remove member
-Opacity of workspace section fixed
-Member name captalized

makstackavenue9456 committed:
Merge branch 'fixIssues/resource-allocation' into 'development'

FixIssues/resource allocation

See merge request stackavenue/dailyploy-web!108

iphalgun committed:
-Project name and members name made responsive
-Fixed vertical scroll- members

makstackavenue9456 committed:
Merge branch 'fixIssues/allocation' into 'development'

FixIssues/allocation

-Project name and members name made responsive and if extends the limit "...".
-Vertical scroll-members now appear behind the project header bar.
-Adding and removing member problem fixed

See merge request stackavenue/dailyploy-web!109

makstackavenue9456 committed:
Merge branch 'fix/allocation-table' into 'development'

fix/allocation-table

See merge request stackavenue/dailyploy-web!111

iphalgun committed:
fix/allocation-table

makstackavenue9456 committed:
Merge branch 'fix/UI-issues' into 'development'

Fix/UI-Issues

See merge request stackavenue/dailyploy-web!112

iphalgun committed:
-allocation table 10*9
-Added loader when we switch projects in roadmap
-On tab now works on roadmap task addition
-add task field will be closed on esc button

iphalgun committed:
-Fixed edit task functionality in roadmap

makstackavenue9456 committed:
Merge branch 'fix/editTask-roadmap' into 'development'

-Fixed edit task functionality in roadmap

See merge request stackavenue/dailyploy-web!114

makstackavenue9456 committed:
-Corrected Email config Logic
- Float Right button

makstackavenue9456 committed:
Merge branch 'fix/email-signup' into 'development'

email config

See merge request stackavenue/dailyploy-web!115

makstackavenue9456 committed:
- CHanges done to hard code email text in configure email in workspace

makstackavenue9456 committed:
Merge branch 'fix/email-signup' into 'development'

Email signup

See merge request stackavenue/dailyploy-web!116

geektj committed:
- font-color change in general settings

geektj committed:
- add color used in settings background

geektj committed:
- import colors file used bg colour and height fixes in workspaces settings tabs full height show

geektj committed:
- email configuration short name box fix to/cc/bcc

geektj committed:
- spinner loader use in user settings workspace name

geektj committed:
- loader add in configure statuses and configure status projects

iphalgun committed:
- Added Userstory component
- Added Userstory Modal component for extra details and
  Userstory card component for showing list of userstory

makstackavenue9456 committed:
Merge branch 'fix/user-story-v2' into 'development'

Fix/user story v2

See merge request stackavenue/dailyploy-web!120

makstackavenue9456 committed:
- changed state toprosp as src in itaskinfomodal

makstackavenue9456 committed:
Merge branch 'hotfix/time-track-edit' into 'development'

- changed state toprosp as src in itaskinfomodal

See merge request stackavenue/dailyploy-web!123

geektj committed:
- merge conflict resolve in user settings file

geektj committed:
- loader add in categories settings

geektj committed:
- semi-colon add in imported file

geektj committed:
- save-btn height and alignment fixes in general settings

geektj committed:
- btn height fixes in email-configuration configure btn

makstackavenue9456 committed:
Merge branch 'fix/user-story-v3' into 'development'

Fix/user story v3

See merge request stackavenue/dailyploy-web!124

iphalgun committed:
- Added Userstory component
- Added Userstory Modal component for extra details and
  Userstory card component for showing list of userstory

iphalgun committed:
- User story deletion integrated

geektj committed:
- condition changes if task categories is null

geektj committed:
Merge branch 'development' of gitlab.com:stackavenue/dailyploy-web into bugfix/Settings

geektj committed:
- click on suspend btn modal box fixes

geektj committed:
- click on save btn multiple time calling api disabled and onchange event in gen settings

geektj committed:
- on click of btn multiple times api calls disabled

geektj committed:
- search bar shift right side and expanded on click

makstackavenue9456 committed:
Merge branch 'feature/user-story-v4' into 'development'

Delete user story

See merge request stackavenue/dailyploy-web!125

makstackavenue9456 committed:
Merge branch 'feature/add-currency-option' into 'development'

Feature/add currency and expense option

See merge request stackavenue/dailyploy-web!82

Jeshan7 committed:
Added currency field in workspace settings
  Added monthly and hourly expense field in add member modal

geektj committed:
- icon changes in configure status settings

geektj committed:
- flag maintain for loader after api calls

geektj committed:
- deleted one col num. of task add icon and box width fix

geektj committed:
- merge conflict resolve in general settings

geektj committed:
- merge conflict resolve accept incoming changes

makstackavenue9456 committed:
Merge branch 'enhancement/roadmap_status' into 'development'

Enhancement/roadmap status

See merge request stackavenue/dailyploy-web!126

iphalgun committed:
- Added Userstory component
- Added Userstory Modal component for extra details and
  Userstory card component for showing list of userstory

iphalgun committed:
- Check added in CommentUpload cmp
- Remove uploaded image functionality fixed/added

makstackavenue9456 committed:
Merge branch 'fix/comments' into 'development'

fix/comments

See merge request stackavenue/dailyploy-web!127

makstackavenue9456 committed:
Merge branch 'enhancement/unselected-task-overlay' into 'development'

Enhancement/unselected task overlay

See merge request stackavenue/dailyploy-web!128

Jeshan7 committed:
Merge branch 'development' of https://gitlab.com/stackavenue/dailyploy-web into feature/userstory-attachments

geektj committed:
Bugfix/settings

makstackavenue9456 committed:
Merge branch 'bugfix/Settings' into 'development'

Bugfix/settings

See merge request stackavenue/dailyploy-web!121

geektj committed:
- add workspace text under workspace box

makstackavenue9456 committed:
- roadmmap goals on roadmap
- configure workflow

makstackavenue9456 committed:
Merge branch 'hotfix/cosmetic-label' into 'development'

- roadmmap goals on roadmap

See merge request stackavenue/dailyploy-web!129

geektj committed:
- add date created in categories in date of creation

geektj committed:
- search filter UI fixes in reports section

geektj committed:
- merge conflict resolve in search filter and categories settings

geektj committed:
- UI fixes in search filter in reports

iphalgun committed:
- Filter by member and status functionality added
- Summary details of member and status

makstackavenue9456 committed:
Merge branch 'enhancement/roadmap_summary' into 'development'

Enhancement/roadmap summary

See merge request stackavenue/dailyploy-web!130

makstackavenue9456 committed:
Merge branch 'bugfix/Settings' into 'development'

search filter UI cosmetics fixes

See merge request stackavenue/dailyploy-web!131

makstackavenue9456 committed:
- Summary filter conjunction

makstackavenue9456 committed:
Merge branch 'enhancement/summary-filter' into 'development'

- Summary filter conjunction

See merge request stackavenue/dailyploy-web!132

iphalgun committed:
- Remove admin modal UI fixed
- Add admin modal position fixed

makstackavenue9456 committed:
Merge branch 'fix/removeAdmin-modal' into 'development'

Remove admin modal UI fixed

See merge request stackavenue/dailyploy-web!133

makstackavenue9456 committed:
Merge branch 'hotfix/time-track-issues' into 'development'

Hotfix/time track issues

See merge request stackavenue/dailyploy-web!134

makstackavenue9456 committed:
- CHanged Calulation of time spent
- Hidden User story
- CHanged the roadmap Status

makstackavenue9456 committed:
Merge branch 'enhancement/roadmap-filter' into 'development'

Enhancement/roadmap-filter

See merge request stackavenue/dailyploy-web!135

iphalgun committed:
- Filter member search bar added
- Multiple member selection functionality added

iphalgun committed:
- Filter member search bar added
- Multiple member selection functionality added

makstackavenue9456 committed:
Merge branch 'enhancement/user-story-filter' into 'development'

Enhancement/userStoryFilter

See merge request stackavenue/dailyploy-web!136

makstackavenue9456 committed:
Merge branch 'fixes/roadmap-issues' into 'development'

Fixes/roadmap issues

See merge request stackavenue/dailyploy-web!137

iphalgun committed:
- Fixed, first time roadmap visit gives internal server error
- On logout, local storage will be cleared

iphalgun committed:
- Roadmap filter loader fixed
- Default loader position fixed
- Added toast message for errors

makstackavenue9456 committed:
Merge branch 'fixes/roadmap-issues-v2' into 'development'

Fixes/roadmap issues v2

See merge request stackavenue/dailyploy-web!139

makstackavenue9456 committed:
- Bugs fix
- Admin option
- Barchart hover

makstackavenue9456 committed:
Merge branch 'Feature_UI/Analysis-Tab' into 'development'

Feature ui/analysis tab

See merge request stackavenue/dailyploy-web!138

Jeshan7 committed:
- Bugs fix
- Admin option
- Barchart hover

makstackavenue9456 committed:
Merge branch 'Feature_UI/Analysis-Tab' into 'development'

Feature UI/analysis tab

See merge request stackavenue/dailyploy-web!141

makstackavenue9456 committed:
- Added g Tag in dp Application

makstackavenue9456 committed:
Merge branch 'feature/g-analytics' into 'development'

- Added g Tag in dp Application

See merge request stackavenue/dailyploy-web!143

makstackavenue9456 committed:
-Added support option

makstackavenue9456 committed:
Merge branch 'feature/g-analytics' into 'development'

-Added support option

See merge request stackavenue/dailyploy-web!144

makstackavenue9456 committed:
Merge branch 'bugFix/dashboardUI' into 'development'

Status and log time issue Fix

See merge request stackavenue/dailyploy-web!147

awesomearpit committed:
Status and log time issue Fix

Jeshan7 committed:
- Bugs fix
- Admin option
- Barchart hover

makstackavenue9456 committed:
Merge branch 'Feature_UI/Analysis-Tab' into 'development'

Feature ui/analysis tab

See merge request stackavenue/dailyploy-web!142

makstackavenue9456 committed:
Merge branch 'fix/roadmap-issues' into 'development'

Fix/roadmap issues

See merge request stackavenue/dailyploy-web!148

makstackavenue9456 committed:
- All instances get marked on
 mark complete in dashboard tasks

makstackavenue9456 committed:
- loader and check when status not there

makstackavenue9456 committed:
Merge branch 'fix/task-status' into 'development'

- loader and check when status not there

See merge request stackavenue/dailyploy-web!149

Jeshan7 committed:
priority color changes

Jeshan7 committed:
minor priority color changes

Jeshan7 committed:
Merge pull request #2 from StackAvenue/enhance/color-changes

Priority color changes

Jeshan7 committed:
Added unique identifier for dashboard's task

Jeshan7 committed:
minor change in col size in dashboard event

Jeshan7 committed:
minor change in condition for identifier

Jeshan7 committed:
Merge pull request #3 from StackAvenue/enhance/add-identifier

Added unique identifier for dashboard's task

Jeshan7 committed:
added margin in identifier

Jeshan7 committed:
Merge branch 'production' of github.com:StackAvenue/dailyploy-web into enhance/add-identifier

Jeshan7 committed:
Merge pull request #4 from StackAvenue/enhance/add-identifier

Enhance/add identifier

joshinehajoshi committed:
proptypes added in routesjs, Signupjs, workspacejs

joshinehajoshi committed:
propTypes added with optional and required

joshinehajoshi committed:
Added propTypes in all required files

joshinehajoshi committed:
debounce added to onClick of buttons

joshinehajoshi committed:
debounce const added for 500
